CRM/Contact/Page/DedupeFind.php

```diff
@@ -62,8 +62,13 @@ public function run() {
     $context = CRM_Utils_Request::retrieve('context', 'String', $this);
     $limit = CRM_Utils_Request::retrieve('limit', 'Integer', $this);
     $rgid = CRM_Utils_Request::retrieve('rgid', 'Positive');
-    $urlQry = "reset=1&rgid={$rgid}&gid={$gid}&limit={$limit}";
-    $this->assign('urlQuery', $urlQry);
+    $urlQry = array(
+      'reset' => 1,
+      'rgid' => $rgid,
+      'gid' => $gid,
+      'limit' => $limit
+    );
+    $this->assign('urlQuery', implode('&', $urlQry));
```

Comment:
This will assign only **values** of `$urlQry` -- `1&$rgid&$gid&$limit` which is completely different from the one assigned before. This breaks all the button URLs on the merge page.